Source/WebCore/platform/graphics/ColorSerialization.cpp

```diff
@@ -378,6 +378,9 @@ String serializationForRenderTreeAsText(const ExtendedSRGBA<float>& color, bool)
 
 String serializationForCSS(const HSLA<float>& color, bool useColorFunctionSerialization)
 {
+    if (useColorFunctionSerialization)
+        return serializationForCSS(convertColor<ExtendedSRGBA<float>>(color), useColorFunctionSerialization);
+
     return serializationForCSS(convertColor<SRGBA<uint8_t>>(color), useColorFunctionSerialization);
```

Comment:
A little strange to be passing the boolean in for functions on both sides of the `if` statement. I’d be tempted to write `true` and `false`.

Reply:
Yeah, that is dumb. Will fix there and in a few other places.